tests/jballerina-integration-test/src/test/resources/configurables/configStructuredTypesProject/configStructuredTypes/main.bal

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+
+import configStructuredTypes.mod1;
+import ballerina/jballerina.java;
+import configStructuredTypes.mod2;
+import testOrg/configLib.mod1 as configLib;
+import ballerina/test;
+
+public type Engineer record {|
+    string name = "";
+    int id = 555;
+|};
+
+configurable Engineer & readonly 'engineer = ?;
+configurable configLib:Manager & readonly manager = ?;
+configurable mod1:Student & readonly student = ?;
+configurable mod1:Person & readonly person = ?;
```

Comment:
Shall we have variables with following types as well?
```ballerina
configurable mod1:Person person1 = ?;
configurable mod1:Employee & readonly employee1 = ?;
configurable table<mod1:Person & readonly> & readonly personTable1 = ?;
configurable table<mod1:Employee & readonly> & readonly employeeTable1 = ?;
```

Reply:
Fixed